install_scripts/install_lago.sh

```diff
@@ -39,13 +39,30 @@ install_lago() {
 }
 
 add_lago_repo() {
+    local DISTRO
     if ! grep -q "Fedora" /etc/redhat-release; then
-        yum -y install epel-release
-        local DISTRO=el
+        DISTRO="el"
+        if grep -q "CentOS" /etc/redhat-release; then
```

Comment:
Yep.. had that in mind. Maybe just fail if its not RHEL/CentOS? (have no idea what repos should be configured there).